content/00.front-matter.md

```diff
@@ -12,7 +12,7 @@ _A DOI-citable version of this manuscript is available at <https://doi.org/DOI_H
 <small><em>
 This manuscript
 {% if ci_source is defined -%}
```

Comment:
Because `ci_source.manuscript_url` is created in a separate Manubot function `add_manuscript_urls_to_ci_params`, should we also ensure that `ci_source.manuscript_url`  is defined here?  Or only check for `ci_source.manuscript_url`?

Reply:
I don't think `ci_params` will ever exist without `manuscript_url` with the current manubot software because:

https://github.com/manubot/manubot/blob/1b885c614634d6e88565ebcf5052b6884132ab32/manubot/process/util.py#L193-L196

But it doesn't hurt to be safe and possibly some users will have an outdated `manubot` python package with this version of `content/00.front-matter.md `, so I added the extra check in https://github.com/manubot/rootstock/pull/263/commits/b07d870959f583f0fb6424dd1648d3795411999b